stable/aerospike/templates/statefulset.yaml

```diff
@@ -21,6 +21,10 @@ spec:
       - name: {{ template "aerospike.fullname" . }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command:
```

Comment:
You should probably wrap that into `if` block. Otherwise you'd probably set empty command and args.

Reply:
done along with other suggestions.